src/runner.js

```diff
@@ -137,6 +138,7 @@ exports.run = function(config) {
       }
       if (config.sauce && config.sauce.launchSauceConnect) {
         console.log('Connecting tunnel');
+        config.sauce.tunnelIdentifier = uuidv5(config.sauce.username, uuidv5.URL);
```

Comment:
@xwa130saucer any thoughts on using `shortid`? simply because it's shorter ;)
https://github.com/dylang/shortid

Reply:
For the purpose of running tests, we need to have a seed to generate the same id every time we feed into the same input. Will change username to accessKey for safe.